---
layout: page
permalink: /softwaredata/
title: Software/Data
tagline: Harvey Lab
tags: [about, software, data]
modified: 8-10-20
comments: true
image:
  feature: mauritia.jpg
---
***

We try to make all our data and software freely available and easily accessible. Scripts are generally available from Github, and data from NCBI, Dryad, or elsewhere. If you notice something that is missing, please <a href="mailto:mgh272@gmail.com" target="_blank">contact us</a>.

<br />


# Software
***

# <a href="https://github.com/mgharvey/tyranni" target="_blank">tyranni</a>

Code plus phylogenies, sequence alignments, and other data used to examine diversification in suboscine passerines (Harvey et al. 2020).

# <a href="https://github.com/mgharvey/ES-sim" target="_blank">ES-sim</a>

A simple, simulation-based test for trait-dependent diversification (Harvey and Rabosky 2018).

# <a href="https://github.com/mgharvey/seqcap_pop" target="_blank">seqcap_pop</a>

A pipeline for *de novo* assembly of population-level sequence capture datasets (Harvey et al. 2016, Syst. Biol.).

# <a href="https://github.com/mgharvey/mps-sim" target="_blank">mps-sim</a>

A program for simulating large sets of alignments resembling those obtained from massively parallel sequencing approaches like RAD-Seq and sequence capture of ultraconserved elements.

# <a href="https://github.com/mgharvey/misc_python" target="_blank">misc_python</a>

A repository of miscellaneous Python code, mostly for processing and manipulating genetic data. Many of these are scripts for formatting input files for different phylogenetic and population genetics programs. Included are scripts for:

* <a href="https://github.com/mgharvey/misc_python/blob/master/bin/mean_distances_from_alignments.py" target="_blank">Calculating pairwise genetic distances</a> between individuals using many alignments of sequence data (loci)

* <a href="https://github.com/mgharvey/misc_python/tree/master/bin/TreeMix" target="_blank">Formatting input files for Treemix</a> (Pickrell and Pritchard 2012) and bootstrapping Treemix trees

* <a href="https://github.com/mgharvey/misc_python/tree/master/bin/BUCKy" target="_blank">Formatting input files for BUCKy</a> (Larget 2010) and running MrBayes (Huelsenbeck and Ronquist 2001) and mbsum iteratively on many loci

* <a href="https://github.com/mgharvey/misc_python/tree/master/bin/dadi" target="_blank">Formatting input files for ∂a∂i</a> (Gutenkunst 2009) and also some example scripts for demographic models, running ∂a∂i, obtaining summary statistics, reformatting output, etc.

* <a href="https://github.com/mgharvey/misc_python/blob/master/bin/fill_in_missing_taxa.py" target="_blank">Detecting missing taxa in alignments</a> (in a directory with many alignments) and adding in missing data for those taxa

* <a href="https://github.com/mgharvey/misc_python/blob/master/bin/split_diplotypes_from_nexus.py" target="_blank">Splitting hapolotypes with ambiguity codes</a> at heterozygous sites into diplotypes

# <a href="https://github.com/mgharvey/misc_R" target="_blank">misc_R</a>

A repository of miscellaneous R code, mostly for analyzing and visualizing data, including scripts for:

* <a href="https://github.com/mgharvey/misc_R/blob/master/R/cloudogram.R" target="_blank">Visualizing rough distributions of gene trees</a> as "cloudograms"

* <a href="https://github.com/mgharvey/misc_R/blob/master/R/map_results_plot.R" target="_blank">Plotting Blastn results on a reference</a>

<br />

# Data
***

* <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA655842" target="_blank">Raw reads </a> and <a href="https://github.com/mgharvey/tyranni" target="_blank">sequence alignments </a>from sequence capture in 1940 suboscine passerines (Harvey et al. 2020, Science)

* <a href="https://www.ncbi.nlm.nih.gov//bioproject/389814" target="_blank">Raw reads from sequence capture in 464 Amazonian birds used for comparative phylogeography of upland and floodplain habitats </a>(Harvey et al. 2017, Am. Nat.)

* <a href="https://github.com/mgharvey/differentiation_speciation" target="_blank">Curated code and data used to compare population differentiation to speciation rates in New World birds </a>(Harvey et al. 2017, PNAS)

* <a href="http://datadryad.org/resource/doi:10.5061/dryad.604b8" target="_blank">Raw reads from sequence capture and RAD-Seq of 8 individuals of *Xenops minutus* </a>(Harvey et al. 2016)

* <a href="http://www.ncbi.nlm.nih.gov/bioproject/PRJNA280209" target="_blank">Short read data from RAD-Seq of 8 individuals in each of 4 bird species (*Rallus crepitans/obsoletus*, *Trochilus polytmus*, *Cranioleuca antisiensis*, and *Xenops minutus*)</a> (Harvey et al. 2015)

* <a href="http://datadryad.org/resource/doi:10.5061/dryad.3j0b1" target="_blank">RAD-Seq genotypes from 72 individuals of *Xenops minutus* </a>(Harvey and Brumfield 2015)

* <a href="http://datadryad.org/resource/doi:10.5061/dryad.qm4j1" target="_blank">UCE alignments from multiple individuals within 5 Neotropical birds</a> (Smith et al. 2014)

* <a href="http://datadryad.org/resource/doi:10.5061/dryad.sd080" target="_blank">UCE alignments from 32 Neoavian birds</a> (McCormack et al. 2013)

* **Field observational data** are available from <a href="https://ebird.org/profile/NTE3MDc/world" target="_blank">eBird</a> and the <a href="http://www.avianknowledge.net/" target="_blank">Avian Knowledge Network</a>

* **Audio recordings, images, and video from the field** are available from the <a href="https://search.macaulaylibrary.org/catalog?view=grid&userId=USER51707" target="_blank">Macaulay Library</a>

* **Specimens from field work** are deposited at the following institutions:
    - <a href="http://www.utep.edu/biodiversity/" target="_blank">Biodiversity Collections at The University of Texas at El Paso</a>   
    - <a href="https://pagina.fciencias.unam.mx/vida-en-ciencias/instalaciones/instalaciones-academicas/museos/museo-zoologia-colecciones" target="_blank">Colecciones del Museo de Zoología Alfonso L. Herrera, UNAM</a>  
    - <a href="https://scientific-collections.gbif.org/institution/68d85ac1-b19a-476e-a63b-0c06e632b7c1" target="_blank">Centre de Recherche en Sciences Naturelles (CRSN/Lwiro)</a>  
    - <a href="http://appl003.lsu.edu/natsci/lmns.nsf/index" target="_blank">Louisiana State University Museum of Natural Science</a>   
    - <a href="http://www.cumv.cornell.edu/" target="_blank">Cornell University Museum of Vertebrates</a>   
    - <a href="http://www.mz.usp.br/" target="_blank">Museu de Zoologia da Universidade de São Paulo</a>  
    - <a href="http://www.corbidi.org/" target="_blank">Centro de Ornitología y Biodiversidad</a>
    - <a href="http://museonoelkempff.org/sitio/index.html" target="_blank">Museo de Historia Natural Noel Kempff Mercado</a>
    - <a href="http://museo.biologia.ucr.ac.cr/" target="_blank">Museo de Zoología de la Universidad de Costa Rica</a>  
    - <a href="http://naturalhistory.ku.edu/" target="_blank">University of Kansas Natural History Museum</a>
    - <a href="https://lsa.umich.edu/ummz" target="_blank">University of Michigan Museum of Zoology</a>
    - <a href="http://asbf16.manas.edu.kg/index.php/en/" target="_blank">Kyrgyz Turkish Manas University</a>
